server/data/incentivesApi.ts

```diff
@@ -207,13 +207,6 @@ export class IncentivesApi extends RestClient {
     })
   }
 
-  /**
-   * @deprecated use getPrisonIncentiveLevels
-   */
-  getAvailableLevels(agencyId: string): Promise<Level[]> {
```

Comment:
the `Leve` interface can be deleted too as it'll be unreferenced

Reply:
Good point, removed here: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/626/commits/002d1c203a1cfadfb1b50d52b444fc5105d730cd